app/src/main/java/org/thoughtcrime/securesms/ui/Colors.kt

```diff
@@ -97,41 +87,37 @@ fun Colors(name: String, colors: List<Color>) {
 @Preview
 @Composable
 fun PreviewThemeColors(
-    @PreviewParameter(ThemeResPreviewParameterProvider::class) themeResId: Int
+    @PreviewParameter(SessionColorsParameterProvider::class) sessionColors: SessionColors
 ) {
-    PreviewTheme(themeResId) {
-        ThemeColors()
-    }
+    PreviewTheme(sessionColors) { ThemeColors() }
 }
 
 @Composable
 private fun ThemeColors() {
-    AppTheme {
-        Column {
-            Box(Modifier.background(MaterialTheme.colors.primary)) {
-                Text("primary")
-            }
-            Box(Modifier.background(MaterialTheme.colors.primaryVariant)) {
-                Text("primaryVariant")
-            }
-            Box(Modifier.background(MaterialTheme.colors.secondary)) {
-                Text("secondary")
-            }
-            Box(Modifier.background(MaterialTheme.colors.secondaryVariant)) {
-                Text("secondaryVariant")
-            }
-            Box(Modifier.background(MaterialTheme.colors.surface)) {
-                Text("surface")
-            }
-            Box(Modifier.background(MaterialTheme.colors.primarySurface)) {
-                Text("primarySurface")
-            }
-            Box(Modifier.background(MaterialTheme.colors.background)) {
-                Text("background")
-            }
-            Box(Modifier.background(MaterialTheme.colors.error)) {
-                Text("error")
-            }
+    Column {
+        Box(Modifier.background(MaterialTheme.colors.primary)) {
+            Text("primary", style = MaterialTheme.typography.base)
```

Comment:
Shouldn't you use our Session colours? 
If we have a fully custom theme we can phase out all the MaterialTheme references.

Reply:
done.